workflows/bigstitcher.nf

```diff
@@ -9,11 +11,21 @@ module_params = params.module_params
 
 if (params.xml) {
     xml_file = file("${params.xml}")
-} 
-else { 
+} else {
     xml_file = null 
 }
 
+if (params.output ) {
+    if (params.output.startsWith('s3://') ||
+        params.output.startsWith('gs://') ||
+        params.output.startsWith('https://')) {
+        output = params.output
+    } else {
+        output = file(params.output)
+    }
+} else {
+    output = file(params.outdir)
```

Comment:
If we're using `output` instead of `outdir`, then I would just remove `outdir` from the code. This kind of stuff just adds complexity IMO.